samples/excel/99-just-for-fun/gradient.yaml

```diff
@@ -8,26 +8,27 @@ api_set:
     ExcelApi: '1.4'
 script:
     content: |-
-        initializeColorPickers();
+        // initializeColorPickers();
```

Comment:
Since we removed this function, should we just delete this line of code?

Reply:
Thanks for catching this. I had commented it out during testing and forgot to remove it.